rtf

```diff
@@ -137,35 +137,40 @@ class Vagrant(Host):
   def reset_platform(name, hosts):
     """ Update or replace the Vagrantfile configuration for the given platform """
     lines = []
+    # parse Vagrantfile into the lines array (and update it)
     with open("Vagrantfile", "r+") as fd:
       updated = False
```

Comment:
Just a note/question here, could we replace "updated" variable  by a "break" at the end of the while ?  or check updated in the while function?